jib-plugins-common/src/main/java/com/google/cloud/tools/jib/plugins/common/JavaContainerBuilderHelper.java

```diff
@@ -95,7 +95,18 @@ public static JibContainerBuilder fromExplodedWar(
       javaContainerBuilder.addClasses(webInfClasses, isClassFile);
     }
     if (Files.exists(webInfLib)) {
-      javaContainerBuilder.addDependencies(new DirectoryWalker(webInfLib).filterRoot().walk());
+      javaContainerBuilder.addDependencies(
+          new DirectoryWalker(webInfLib)
+              .filterRoot()
+              .filter(path -> !path.getFileName().toString().contains("SNAPSHOT"))
+              .walk());
+    }
+    if (Files.exists(webInfLib)) {
```

Comment:
This second `if` isn't necessary. Just put `.addSnapshotDependencies` into the above `if` body.

Reply:
right, oops.